src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -95,7 +96,12 @@ public void run() {
         }
         // Loop Syncthing
         Process process = null;
+        // Potential fix for #498, keep the CPU running while native binary is running
+        PowerManager pm = (PowerManager) mContext.getSystemService(Context.POWER_SERVICE);
+        PowerManager.WakeLock wakeLock = useWakeLock() ? pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, TAG) : null;
```

Comment:
Can you put the `?` and the `:` from the ternary operator on seperate lines? Makes it a bit easier to read.